packages/pages/src/components/image/image.tsx

```diff
@@ -34,34 +28,43 @@ export const Image = ({
   style = {},
 }: ImageProps) => {
   const imgRef = useRef<HTMLImageElement>(null);
-  const [imgStatus, setImgStatus] = useState<imgLoadingStatus>(
-    imgLoadingStatus.NONE
-  );
+  const [imgLoaded, setImgLoaded] = useState(false);
 
   useEffect(() => {
     if (imgRef.current?.complete) {
-      setImgStatus(imgLoadingStatus.LOADED);
+      setImgLoaded(true);
     }
   }, []);
 
-  if (layout != ImageLayout.FIXED && (width || height)) {
-    console.warn(
-      "Width or height is passed in but layout is not fixed. These will have no impact. If you want to have a fixed height or width then set layout to fixed."
-    );
-  }
+  validateRequiredProps(
+    layout,
+    image.image.width,
+    image.image.height,
+    width,
+    height,
+    aspectRatio
+  );
 
-  const imgWidth: number = image.image.width;
-  const imgHeight: number = image.image.height;
+  const imgWidth: number = Math.abs(image.image.width);
+  const imgHeight: number = Math.abs(image.image.height);
   const imgUUID = getImageUUID(image.image.url);
 
+  // The image is invalid, only try to load the placeholder
+  if (!imgUUID) {
+    return <>{placeholder != null && placeholder}</>;
+  }
+
+  const absWidth = width ? Math.abs(width) : undefined;
```

Comment:
Why is this needed? Validation should handle it. It's confusing to validate something, then also fix it and use some other variable here.

Reply:
hmm, the same logic is used throughout the file (where even if you have an invalid config, we still try to render an image to our best effort. For example, if you have layout = "aspect" but do not provide a ImageProps.aspectRatio, we still try to render an image with the `aspectRatio = ImageProps.image.width/ImageProps.image.height`.  Do we want to change that?

Reply:
This is probably a call for Max. Idk if "try to render it as best we can" is good or if it leads to unknown outcomes. We can go with "try our best" for now, but can you start a thread about it (tomorrow is fine) and cc me?

Reply:
looks like Max wants to go with the current way